src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -490,6 +492,20 @@ struct Query {
   7: optional FieldQuery       field
 }
 
+struct AggregateTilesRequest {
+	1: required string sourceNameSpace
+	2: required string targetNameSpace
```

Comment:
nit: Use "sourceNamespace" and "targetNamespace". We don't camel case namespace. (with the exception of when it's just "nameSpace" in thrift definitions, and this is only done because "namespace" not-camel cased is reserved by thrift....)

Reply:
It's very frequently used as a whole word:
https://www.w3.org/2001/03/identification-problem/namespace_as_a_word.html

Reply:
Done.